package.json

```diff
@@ -24,9 +24,9 @@
     "lint:fix": "npm run lint:fix --workspaces --if-present",
     "build": "npm run build --workspaces --if-present",
     "test:watch": "npm run test:watch --workspaces --if-present",
-    "patch": "npm version patch -w",
-    "minor": "npm version minor -w",
-    "major": "npm version major -w",
+    "patch": "npm version patch -w && npm outdated",
```

Comment:
I think outdated needs to go first because the version commands take an argument :)

Reply:
oh sorry I don't know about that